internal/utils/utils.go

```diff
@@ -28,3 +30,22 @@ func ErrorString(e error) string {
 	}
 	return e.Error()
 }
+
+var IsVerbose bool
+
+// Conditionally (format) print verbose messages
+func Verbosef(msg string, args ...any) {
+	if IsVerbose {
+		fmt.Printf(msg+"\n", args...)
+	}
+}
+
+// Testfunction to test logLevel setting
```

Comment:
This function should be removed?

Reply:
Eventually? Let's keep it there, it's only invoked on -d now (and thus, debugs the logging for us) since we might want to play with levels later?

Reply:
I don't think it's useful, debugging levels should be done on the initial implementation which works already right?

Reply:
I meant we might want to add a value to -d later, to set different levels, so this is there when we want to test that? I am in favor of removing dead code, but It's not hurting us right now, is it?